app.rb

```diff
@@ -143,7 +143,7 @@ def quality_indicator_group input
     @quality, response_code = PodQualityEstimate.load_quality_estimate(@name)
     @page_title = "#{@name}'s Quality Estimate on CocoaPods.org"
 
-    if response_code == 404
+    if (400...600).cover? response_code
       not_found
```

Comment:
Should we really be returning a not found error when we don't actually know if it's not found?